common/versioning/version.properties

```diff
@@ -1,3 +1,3 @@
 #Thu Aug 02 12:03:16 PDT 2018
-versionName=3.0.2
+versionName=3.0.3-RC1
```

Comment:
We are doing another release right now, just for this?

Reply:
Yes, otherwise we'll lose a lot of logs in Broker due to this noise.
I'm not planning to publish this to maven repo though.

Reply:
Okay